src/System.Management.Automation/engine/Interop/Windows/OpenProcessToken.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Windows
+    {
+        internal const int TOKEN_QUERY = 0x0008;
+
+        [LibraryImport("api-ms-win-core-processsecurity-l1-1-0.dll",  SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static partial bool OpenProcessToken(nint ProcessHandle, int DesiredAccess, out nint TokenHandle);
```

Comment:
Up to you but there's the [TokenAccessLevels enum](https://learn.microsoft.com/en-us/dotnet/api/system.security.principal.tokenaccesslevels?view=net-7.0) that map to DesiredAccess here. No need for the int and manual const to get this working.